operations/helm/charts/grafana-agent/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: grafana-agent
 description: "Grafana Agent"
 type: application
-version: 0.2.3
```

Comment:
why 3.0 and not 2.4? Just curious.

Reply:
I did 0.2.4 at first but then I realized if you're setting `configReloader.customArgs` in the values, this is a breaking change, which (to me) meant a minor version bump since the public interface of the values.yaml changed.